.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,55 @@
-Make sure you are running the latest version of Home Assistant before reporting an issue.
-
-You should only file an issue if you found a bug. Feature and enhancement requests should go in [the Feature Requests section](https://community.home-assistant.io/c/feature-requests) of our community forum:
+<!-- READ THIS FIRST:
+- Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
+- Do not report issues for components if you are using custom components: files in <config-dir>/custom_components
+- Frontend (user interface of Home Assistant in your browser) related issues have to be reported here: https://github.com/home-assistant/home-assistant-polymer/issues
+- This is for bugs only. Feature and enhancement requests should go in our community forum: https://community.home-assistant.io/c/feature-requests
+- Answer all question as detailed as possible, write the information inside the backticks. Do not delete any text from this template!
+
+-->
+**1. Home Assistant version:**
+```
 
-**Home Assistant release (`hass --version`):**
+```
+<!--
```

Comment:
I think this comment should be before the backticks; at least that would make more sense to me.